src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -64,20 +63,23 @@ class FormStartControls extends React.Component {
   };
 
   render() {
+    const { formConfig } = this.props.routes[1];
+    const startNewAppMessage =
+      formConfig.startNewAppMessage || 'Start a new application';
     if (this.props.formSaved) {
       return (
         <div>
           {!this.props.isExpired && (
             <ProgressButton
               onButtonClick={this.handleLoadForm}
-              buttonText="Continue your application"
+              buttonText={startNewAppMessage}
```

Comment:
This previously said "Continue your application", not "Start your application". Looks like we need to pull in the `continueAppMessage` in here too.

Reply:
Fixed